threefive/version.py

```diff
@@ -1,30 +1,34 @@
 """
-threeefive.version
+threefive.version
 
-Odd number versions are releases.
-Even number versions are testing builds between releases.
-
-Used to set version in setup.py
-and as an easy way to check which
-version you have installed.
+Note: this is an update to the way that the version of Threefive is set.
+Originally, the major, minor and patch numbers were defined in this file and
+then concatenated to form the version, which was then used in the setup.py
+file.
+However, this led to a build-time error since the threefive module was being
+used before it was built. In order to get around this, the version is now set
+from a VERSION file found in the root of the module, following the same
+convention as before, e.g. 2.3.75.
+It is this VERSION file that needs to be updated when the version of threefive
+is to be incremented.
+These methods are just for convenience and for backwards compatibility with
+any current consumer of the version module.
 """
 
-MAJOR = "2"
-MINOR = "3"
-MAINTAINENCE = "75"
-
 
-def version():
+def version() -> str:
```

Comment:
I have been expecting somebody to push me toward the newer style functions, I need to start using them.